### PR TITLE
TNO-2370 Add media type to Program/Show

### PR DESCRIPTION
**Pull request by fbarreta**
Add relationship between Program/Shows (Series).

- Migration
- Service
- Frontend

### CODE REVIEW

File: libs/net/dal/Services/SeriesService.cs [hunk 1]
```diff
@@ -83,9 +86,55 @@ public IPaged<Series> Find(SeriesFilter filter)
...
+        entity.MediaTypeSearchMappingsManyToMany.ForEach(a =>
+            {
+                var originalMedia = originalMedias.FirstOrDefault(rs => rs.MediaTypeId == a.MediaTypeId);
+                if (originalMedia == null)
+                {
+                    a.MediaType = this.Context.MediaTypes.FirstOrDefault(x => x.Id == a.MediaTypeId);
+                    original.MediaTypeSearchMappingsManyToMany.Add(a);
+                }
+            });
+        return base.UpdateAndSave(original);
```
Comment (Fosol) [inline libs/net/dal/Services/SeriesService.cs:135]: Your saving the original without apply any of the other changes that may have occurred to the series.

Comment (fbarreta) [reply libs/net/dal/Services/SeriesService.cs:135]: my bad ... ;)
updating

**Review by Fosol — COMMENTED** [review]
I think we just need a new tno-core published?

**Comment by fbarreta** [comment]
> I think we just need a new tno-core published?

Yeah. Once it's done, I can wrap up.

**Comment by Fosol** [comment]
I have published `tno-core:0.1.36` from this branch.  I have rebased and pushed it back up.  Please pull down and reset your branch appropriately to test and verify before we merge.

**Pull request merged**